modules/xmpp/ChatRoom.spec.js

```diff
@@ -1,9 +1,13 @@
-import { parser } from './ChatRoom';
+import ChatRoom, { parser } from './ChatRoom';
 import { $pres } from 'strophe.js';
+import XMPPEvents from '../../service/xmpp/XMPPEvents';
 
 // This rule makes creating the xml elements take up way more
 // space than necessary.
 /* eslint-disable newline-per-chained-call */
+// These rules makes the xml strings harder to read
+/* eslint-disable operator-linebreak */
+/* eslint-disable max-len */
```

Comment:
nit: you can combine the ignored rules onto one line, `eslint-disable rule, rule, rule`.

Reply:
fixed